### ISSUE TITLE
Kodadot Home UX

### ISSUE THREAD

**Issue by akudless** [kodadot/nft-gallery#225]
I had some ideas about the main gallery page to help make it a bit more informative. I know many of these ideas are being worked on (or have already been updated since I made this yesterday) and are listed in other issues, but sometimes its helpful to look at the big picture. This builds on Issue #60. I'm a designer/artist so I made a quick mockup first and then wrote this stuff to explain my thoughts.
![Kodadot_UX_300](https://user-images.githubusercontent.com/11219277/112353952-3e1ef300-8c9a-11eb-8711-2aa260adb7a8.jpg)

**Thumbnail Preview**
At the moment, each preview only shows the NFT's title.  It would be cool if we had the following:

- Artist Name: This could be the pseudonym identity (if the artist has one) or the address. Maybe the address only gets shown if there is a pseudonym as an incentive, but even showing the address gives good incentive to the artist to verify their identity.
- Edition Number: I think this may have already been solved this morning! Great work! This greatly helps declutter the interface with what looks like duplicates.
- Price: If the NFT is for sale, it lists the sale price in KSM and USD (maybe there is a setting for this tied to the user's profile so that users always see the conversion in their preferred currency). If NFT isn't for sale, this field is blank. 

**Search/Filter Menus**
It would be great to be able to filter/sort the gallery (Issues #35 & #49) by things like:

- Recently Sold
- For Sale
- Patronage
- Editions of 1
- Price
- File Type

Or to search the Gallery by:

- Tags
- Artist Name
- Color
- File Type

**Top Left Menu**
I put more externally focused major things here:

- Gallery: Brings you back or reloads the main Gallery
- Artist: Views Gallery based on collections by Artist (Issue #157). Lists one image per artist. This would be defined by the info listed in their profile:

> - image that represents their entire work
> - metadata pulled from pseudonym identity like social links and name (Issue #134 )
> - others that could simply be text fields like a bio or artist statement
> - List of NFTs by the artist sorted by collection

On each Artist preview thumbnail, it could show the chosen header image for that artist and maybe metadata about the work like the number of minted pieces, number sold, number currently for sale, etc. so users get a quite overview of the artist body of work (are they a longtime users with 100s of sold pieces or are they new to the market). This page could also be filtered/sorted so users could quickly discover new artists or artists that share certain tags.

- Create: Brings users to existing Create page that clearly explains the creation process with links to helpful guides/videos. 

**Top Right Menus**
I thought of these being more internally focused. 

- Profile: This would be the Profile of the user (artist/buyer/both). You could see your own collection of minted work or collection of work you have bought. And this is where you could update all of the artist metadata (bio, statement).
- Buy $KSM: quick way to buy KSM via credit card. I know this is already (partially) working, but it's a bit hidden. Since this market specifically and the larger NFT space is still new for lots of people, having a quick and very visible way to add funds and buy seems like a useful link that should get top billing. Even better would be adding credit purchase directly into buying (Issue #162).
- About Kodadot: Backstory of Kodadot, links to tutorials, Github, Medium, FAQ on low-carbon minting, etc.

**Comment by yangwao** [comment]
Hey as a community-owned project, we are trying to do stuff in public manners so it's inviting everyone can participate and add their performance and build a whole better picture, together.

You can make an issue here https://github.com/kodadot/nft-gallery/issues/new, put a screenshot of pdf and I'll put my, same reaction there.
We really appreciate your effort:)

Your take is really insightful for us as you are coming from outside of our brains. For grouping multiple NFTs, we already have issues and it should be on production https://github.com/kodadot/nft-gallery/issues/201 once PR is approved.

Speaking on search & filter - https://github.com/kodadot/nft-gallery/issues/49 & https://github.com/kodadot/nft-gallery/issues/35
We've made major changes to how our DB works in users browsers so it should be a breeze now to add that "views" on NFTs :)

Speaking on NFT detail, adding author may be good, we had it there, but we will for example plan to show only authors with pseudonyms set to sort of incentivizing them for outside attention ability to gain reputation. 

BUY $KSM - Credit was already in navbar but we hide it, but we could put it out, again :)

About section - that's very appealing where we could potentially put all readings and filter out stuff from the landing page

Profile - it's something more complex as we want to enable users to interact with KodaDot without even installing any extension, it will be probably, big. https://github.com/kodadot/nft-gallery/issues/137#issuecomment-804174622

Navbar Artists - what should be there? Profiles of artists?

We really appreciate your feedback and as a gesture of thank you, please post your KSM address as we value your deep and constructive input!

I've [personally rewarded](https://kusama.subscan.io/extrinsic/6755426-1) @akudless for his effort to deep-dive on our UX. :)
Inspiration for others! 🥳

**Comment by Saralamusa** [comment]
I love all this ideas!

**Comment by vikiival** [comment]
Wow seeing these old images of KodaDot brings a lot of memories 🥲 .

I would close this as we have already done a redesign. 

and the rest of the issue is basically #348.

What do you thing @yangwao ?

**Comment by yangwao** [comment]
I guess @akudless could reach out to DM me on our Discord https://discord.gg/35hzy2dXXh yangwao and we'd love him on board, so he might help us on UI/UX as we have a lot of stuff to do there on end-user experience. 
Most of the stuff in this issue is already done and there are some good bites hidden between lines that are worth create new issues for each. Otherwise good to let go.
